score-v1b1.json

```diff
@@ -305,6 +305,11 @@
               "description": "The resource in the target environment.",
               "type": "string"
             },
+            "class": {
+              "description": "A specialisation of the resource type.",
```

Comment:
I didn't add anything about empty class means `default` as it sounds too Humanitec specific to me.